sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com.azure.security.keyvault.administration/BackupAndRestoreHelloWorld.java

```diff
@@ -0,0 +1,61 @@
+        // Instantiate an KeyVaultBackupClient that will be used to call the service. Notice that the client is using
+        // default Azure credentials. To make default credentials work, ensure that environment variables
+        // 'AZURE_CLIENT_ID', 'AZURE_CLIENT_KEY' and 'AZURE_TENANT_ID' are set with the service principal credentials.
+
+        // To get started, you'll need a URI to an Azure Key Vault. See the README (https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Administration/README.md)
+        // for links and instructions.
+        KeyVaultBackupClient backupClient = new KeyVaultBackupClientBuilder()
+            .vaultUrl("https://{YOUR_VAULT_NAME}.vault.azure.net")
+            .credential(new DefaultAzureCredentialBuilder().build())
+            .buildClient();
+
+        // Using the KeyVaultBackupClient, you can back up your entire collection of keys. The backing store for full
+        // key backups is a blob storage container using Shared Access Signature authentication. For more details on
+        // creating a SAS token using the `BlobServiceClient`, see the Azure Storage Blobs client README (https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/README.md)
+        // and the authentication samples(https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/samples/Sample02_Auth.cs).
+        // Alternatively, it is possible to generate a SAS token in Storage Explorer (https://docs.microsoft.com/azure/vs-azure-tools-storage-manage-with-storage-explorer?tabs=windows#generate-a-shared-access-signature-in-storage-explorer).
+
+        // To ensure you have some keys for backup, you may want to first create a key using the KeyClient. To create a
+        // new KeyClient to create a key, see the 'Creating a KeyClient' (https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Keys/samples/Sample1_HelloWorld.md#creating-a-keyclientkeyvault) and [Creating a key](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Keys/samples/Sample1_HelloWorld.md#creating-a-key)
+        // samples.
+
+        // In the sample below, you can set blobStorageUrl and sasToken based on environment variables, configuration
+        // settings, or any way that works for your application.
+        String blobStorageUrl = "<blob-storage-url>";
+        String sasToken = "<sas-token>";
+        SyncPoller<KeyVaultBackupOperation, String> backupPoller = backupClient.beginBackup(blobStorageUrl, sasToken);
+
+        backupPoller.waitForCompletion();
+
+        // Now let's restore the entire collection of keys from the backup. We will need the get the URI for the
+        // location the backup, as well as Shared Access Signature for accessing it.
+        String backupUri = backupPoller.getFinalResult();
+        String[] segments = backupUri.split("/");
```

Comment:
Do we intend to change this to handle the Uri parsing internally? ([.NET example](https://github.com/Azure/azure-sdk-for-net/pull/15885))

Reply:
I think that's probably better for the user experience. I'll set up a separate PR with those changes.

Reply:
Created said PR: #17040.

Reply:
Merged.